asteroid/data/dampvsep_dataset.py

```diff
@@ -0,0 +1,231 @@
+from pathlib import Path
+from multiprocessing import Pool, cpu_count
+from tqdm import tqdm
+import torch.utils.data
+
+import numpy as np
+import pandas as pd
+import random
+import json
+import librosa
+import warnings
+warnings.filterwarnings('ignore')
+
+
+class DAMPVSEPDataset(torch.utils.data.Dataset):
+    """
+    DAMP-VSEP vocal separation dataset
+
+    This dataset treats all songs as SINGLE by converting DUETS into 2 different
+    performances.
+
+    Note: The datasets are hosted on Zenodo and require that users
+          request access.
+          DAMP-VSEP https://zenodo.org/record/3553059
+
+    Args:
+        root_path (str): Root path of dataset
+        csv_dir (str): Path to dataset split in csv format
+        task (str): one of ``'enh_vocal'``,``'enh_both'``
+            * ``'enh_vocal'`` for vocal enhanced.
+            * ``'enh_both'`` for vocal and background separation.
+        samples_per_track (int, optional):
+            Number of samples yielded from each track, can be used to increase
+            dataset size, defaults to `1`.
+        random_segments (boolean, optional): Enables random offset for track segments
+        sample_rate (int, optional): Samplerate of files in dataset.
+            Default 16000 Hz
+        segment (float, optional): Duration of segments in seconds,
+            Defaults to ``None`` which loads the full-length audio tracks.
+        silence_in_segment (float, optional): Max duration of continuous silence in a segment.
+            Default ``None`` which allows any silence length
+        num_workers (int, optional): Number of workers.
+            Default to ``None`` which utilise all available workers
+     """
+
+    dataset_name = 'DAMP-VSEP'
+
+    def __init__(self, root_path, task, split, samples_per_track=1,
```

Comment:
How about having a default for `task` and `split`? 
Positional args are always confusing in the long term.